cfme/configure/access_control.py

```diff
@@ -41,17 +47,73 @@ def simple_user(userid, password):
     return User(name=userid, credential=creds)
 
 
-class User(Updateable, Pretty, Navigatable):
-    user_form = Form(
-        fields=[
-            ('name_txt', Input('name')),
-            ('userid_txt', Input('userid')),
-            ('password_txt', Input('password')),
-            ('password_verify_txt', Input('verify')),
-            ('email_txt', Input('email')),
-            ('user_group_select', AngularSelect('chosen_group')),
-        ])
+class UserForm(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    name_txt = Input(name='name')
+    userid_txt = Input(name='userid')
+    password_txt = Input(id='password')
+    password_verify_txt = Input(id='verify')
+    email_txt = Input('email')
+    user_group_select = BootstrapSelect(id='chosen_group')
+
+    add_button = Button('Add')
```

Comment:
There is no button "Add" in `EditUserView` . It's better to place widgets only in views where are actually presented.